src/components/ArticlesList.jsx

```diff
@@ -0,0 +1,41 @@
+import React, { useEffect } from "react";
+import axios from "axios";
+
+const ArticlesList = ({ getArticles, setGetArticles }) => {
+  useEffect(() => {
+    axios
+      .get("https://news-v9aq.onrender.com/api/articles")
+      .then((response) => {
+        console.log(response.data.articles);
+        setGetArticles(response.data.articles);
+      });
```

Comment:
What about if this request took a really long time to work? Some kind of loading state and message would go a long way for good user feedback